README.md

```diff
@@ -74,7 +74,7 @@ you've configured your node to expose the developer options, e.g.
 ```
 
 #### Mainnet Option
-To test with real bitcoin,  you will need to have a local `bitcoind` node running:
+To test with the  real bitcoin,  you will need to have a local `bitcoind` node running:
```

Comment:
```suggestion
To test with the bitcoin,  you will need to have a local `bitcoind` node running:
```

NIT, IMO it doesn't sound good.

Reply:
Yeah, it's bitcoins the currency, not bitcoin the network / protocol